packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -45,7 +45,7 @@ const getAccountUser = accountSelectors.getAccountUser
 const messages = {
   wasTippedBy: 'Was Tipped By',
   andOthers: (num: number) => `& ${num} ${num > 1 ? 'others' : 'other'}`,
-  sendTipToPrefix: 'SEND TIP TO ',
+  sendTipToPrefix: 'Send Tip to ',
```

Comment:
I'm assuming this was an intentional quick add?

Reply:
yeah basically we dont use this anymore and it was a way to get fully off stems button while still preserving functionality